data/json/vehicles/helicopters.json

```diff
@@ -1,222 +1,389 @@
 [
   {
-    "id": "helicopter_wreck_1a",
+    "id": "helicopter_wreck_1b",
     "type": "vehicle",
-    "name": "Interceptor Wreckage",
+    "name": "AH-64 Apache Wreckage",
```

Comment:
We should avoid using existing names due to the legal reasons.

Reply:
It's not a big deal but probably it'd be better to use more generalized terms as Iv'e made most of those designs pretty generic. Only the Osprey was actually based on a real vehicle (and even here certain liberties were taken due to game's mechanics and limitations), the rest is pretty simplified.

Reply:
> We should avoid using existing names due to the legal reasons.

I emphatically disagree with this.  If it is intended to represent a IRL entity, it should use the name of the IRL entity.

If the intent of the author is to make a "generic attack helicopter", then calling it that is also fine.

Reply:
I'm with Kevin on this.  Plus this is already the case for most non-fictional guns in Cata.  Admittedly, the vehicles are generic, but that I think that has more to do with the lack of "resolution" for smaller vehicles.

Reply:
Then why we did [this](https://github.com/CleverRaven/Cataclysm-DDA/pull/21486)?

Reply:
Because people outside the US don't know wwhat a vacutainer is, so the change was a good one even if prompted by a bogus legal claim?

Reply:
People might as well don't know what does "AH-64 Apache" mean. But in case of vacutainer we have item description, and in this case we don't.

Reply:
Vacutainer was inaccurate, it was being used to refer to the entire blood draw kit, when a vacutaiber is just an ampule that holds the blood.

The Apache is probably the most well known helicopter in the world, plus it doesnt really matter, wreckage is wreckage.

Reply:
>If the intent of the author is to make a "generic attack helicopter", then calling it that is also fine.

Then probably that should go for current things as again, only Osprey was meant to not be generic. Though also again, I don't think it's a huge deal either way, at least as long as no one will push on me to edit my designs later so they'll look more like whatever they'll get called.